src/main/java/nl/rijksoverheid/mev/gezagsmodule/domain/Verblijfplaats.java

```diff
@@ -32,20 +32,27 @@ public String get(final String key) {
         return values.get(key);
     }
 
+    @Override
+    public String get(final String key, final String fieldName) {
+        //inOnderzoek(key, fieldName);
```

Comment:
Commented out code can be removed.

Reply:
heb erboven een referentie toegevoegd naar het issue waar hier naar gekeken moet worden.
Comment laten staan omdat het relevant is te weten waar het probleem nu zit:
https://github.com/BRP-API/brp-api-gezag/issues/17

Reply:
Ah ok, helemaal prima. Dan is het duidelijk dat het gaat om een ongoing issue, in plaats van random commented-out code. :)